### PR TITLE
chore(functions): move helloPubSub to separate directory

### PR DESCRIPTION
**Pull request by ace-n**
Copied from samples in [this directory](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/main/functions/v2).

Those will be deleted once everything is moved over.

Part of b/229394231

### CODE REVIEW

File: functions/v2/helloPubSub/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "nodejs-docs-samples-functions-v2",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
```
Comment (averikitsch) [inline functions/v2/helloPubSub/package.json:6]: nit: Google LLC

File: .github/workflows/functions-v2-hellopubsub.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+name: functions-v2-hellopubsub
```
Comment (averikitsch) [inline .github/workflows/functions-v2-hellopubsub.html:1]: YAML file

**Review by averikitsch — APPROVED** [review]

File: functions/v2/helloPubSub/package.json (hunk 1 above)
Comment (ace-n) [reply functions/v2/helloPubSub/package.json]: Done.

File: .github/workflows/functions-v2-hellopubsub.html (hunk 2 above)
Comment (ace-n) [reply .github/workflows/functions-v2-hellopubsub.html]: Done.

**Review by bcoe — APPROVED** [review]

**Pull request merged**